execute/errors.go

```diff
@@ -0,0 +1,5 @@
+package execute
+
+import "errors"
+
+var errNothingExecuted = errors.New("no reports were executed")
```

Comment:
nit: this name is kind of misleading, it seems to be returned when no messages fit into the report. If I see this in the log I'd think its some sort of transmission error or related. Perhaps `errEmptyReport`?

```golang
var errEmptyReport = errors.New("no messages can fit in the report")
```